commands/monitoring/central/central.go

```diff
@@ -0,0 +1,61 @@
+package central
+
+import (
+	"context"
+	"fmt"
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/ionos-cloud/ionosctl/v6/internal/constants"
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/json2table/jsonpaths"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/jsontabwriter"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/tabheaders"
+	"github.com/ionos-cloud/sdk-go-bundle/products/monitoring/v2"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	allCols = []string{"Id", "Enabled", "GrafanaEndpoint", "Products"}
+)
+
+func CentralCommand() *core.Command {
+	cmd := &core.Command{
+		Command: &cobra.Command{
+			Use:              "central",
+			Aliases:          []string{"c"},
+			Short:            "Central monitoring is a feature that allows you to use other products to send metrics to a central location.",
+			TraverseChildren: true,
+		},
+	}
+
+	cmd.AddCommand(CentralFindByIdCmd())
+	cmd.AddCommand(CentralDisable())
+	cmd.AddCommand(CentralEnable())
+
+	return cmd
+}
+
+func enable(c *core.CommandConfig, enabled bool) error {
+
+	centralId := viper.GetString(core.GetFlagName(c.NS, constants.FlagCentralID))
```

Comment:
remove central ID as this doesn't make sense given the functionality of `/central`